feature-libs/asm/components/asm-components.module.ts

```diff
@@ -22,6 +32,12 @@ import { defaultAsmLayoutConfig } from './default-asm-layout.config';
     I18nModule,
     FormErrorsModule,
     PasswordVisibilityToggleModule,
+    IconModule,
+    PopoverModule,
+    ModalModule,
+    NgSelectModule,
```

Comment:
`PopoverModule`, `ModalModule`, `NgSelectModule`, `PasswordVisibilityToggleModule` and `FormsModule` are not used, right? Please check them.